### ISSUE TITLE
tuxedo

### ISSUE THREAD

**Issue by CKZkkuF6JM** [tuxedocomputers/tuxedo-keyboard#15]
tldr; on clevio laptops, please provide support for color switch mode, not illumination mode by default

I purchased a  TUXEDO InfinityBook Pro 15 with Bungie OS installed by default. This OS provides a full keyboard back-light support out-of the box. In particular, the user can switch between back-light colors with the proper combination of keys (fn + a special key on the num pad).

For personal convenience, I decided to switch to an another OS (Ubuntu based as well), and then lost the back-light support. Fortunately, the tuxedo-keyboard solution provided in this repository added some support for keyboard back-light (and it works well). However, surprisingly after the driver install the combination of fn+key on the num pad doesn't switch between colors anymore, but rather switch between different illuminations modes (flash-light, disco, etc). While IMHO there is a real value in the possible choice between colors on a laptop (eg. high blue color during daytime, and soft red color during nighttime), switching between different frequencies of color change or any "stroboscopic" mode is of lesser interest on a laptop (who can really use a keyboard with continuous back-light flashing?).

Can you please provide help on how to use the fn+num key combination to switch between colors, and not modes, or consider updating your driver accordingly by default?

Thank you very much for your work on this driver.

**Comment by jondo** [comment]
Can this issue please get a more useful title?
